llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -914,6 +966,19 @@ void RISCVFrameLowering::emitPrologue(MachineFunction &MF,
             .addImm(ShiftAmount)
             .setMIFlag(MachineInstr::FrameSetup);
       }
+      if (NeedProbe && RVVStackSize == 0) {
+        // Do a probe if the align + size allocated just passed the probe size
```

Comment:
Is there a test for this? All the tests look like they would have an RVVStackSize != 0.